### ISSUE TITLE
Error while running with specific provision profile in iOS

### ISSUE THREAD

**Issue by iosTeamilink** [NativeScript/ios-runtime#1267]
Hello,

I am facing an issue while running my iOS project in device(iPhone) using command of specific provision profile.

the error is related to SHA-1.

**Unable to apply changes on device: device_ID. Error is: Expected "SHA-1 hash: " or end of input but "S" found..**

I tried basic steps to solve this error. that is to remove platform, hook and node-moudule etc.

Still not solved please help me to solve it ASAP.

Thanks in advance!!

**Comment by mbektchiev** [comment]
Hi,

Are you using the latest version of {N} CLI? It seems there's an old issue related to the same error that you're getting: https://github.com/NativeScript/nativescript-cli/issues/4934

**Comment by iosTeamilink** [comment]
@mbektchiev 

We are using version 5.1.1 of {N} CLI due to this we are facing this issue?

**Comment by mbektchiev** [comment]
Yes, I would strongly encourage you to upgrade your project and use the latest version. There are many other issues and missing features like iOS 13 and macOS Catalina supports in v5.x.
